models/engine/db_storage.py

```diff
@@ -74,3 +75,22 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Get an object"""
+        for clss in classes:
+            if cls is None or cls is classes[clss] or cls is clss:
```

Comment:
multiple variable comparisons can be used with the `in` operator.

ex.
`if cls in (None, classes[clss], clss).`